lib/models/locations_movies.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'locations_movies',
```

Comment:
I think you should be able to get by without a `LocationsMovies` Bookshelf model since the relation in the `Movie` model should implicitly use the `locations_movies` model?